backend/apps/ai/management/commands/ai_create_project_chunks.py

```
@@ -0,0 +1,234 @@
+
+        total_projects = queryset.count()
+        self.stdout.write(f"Found {total_projects} projects to process")
+
+        if total_projects == 0:
+            self.stdout.write("No projects found to process")
+            return
+
+        batch_size = options["batch_size"]
+
+        for offset in range(0, total_projects, batch_size):
+            batch_projects = queryset[offset : offset + batch_size]
```

Flake8 (7.2.0)

[error] whitespace before ':'

(E203)